android/source/VideoLocker/src/org/edx/mobile/view/LoginActivity.java

```diff
@@ -62,17 +62,15 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_login);
-        overridePendingTransition(R.anim.slide_in_from_right,
-                R.anim.slide_out_to_left);
+        overridePendingTransition(R.anim.slide_in_from_bottom, R.anim.no_transition);
 
-        getWindow().setSoftInputMode(
-                WindowManager.LayoutParams.SOFT_INPUT_STATE_HIDDEN);
+        hideSoftKeypad();
```

Comment:
Out of curiosity, why do we need to do this? We show this from the LaunchActivity and there's no keyboard there, right?